components/hab/src/main.rs

```diff
@@ -95,6 +96,30 @@ lazy_static! {
             "group",
         ]
     };
+
+    /// The default filesystem root path to base all commands from. This is lazily generated on
+    /// first call and reflects on the presence and value of the environment variable keyed as
+    /// `FS_ROOT_ENVVAR`.
+    static ref FS_ROOT: PathBuf = {
+        use hcore::fs::FS_ROOT_ENVVAR;
+
+        if cfg!(target_os = "windows") {
```

Comment:
It feels like a minor thing, but I'm sure when people see `cfg!(windows)` in some places and `cfg!(target_os = "windows")` in others, it's going to cause some people confusion.

Reply:
Also, I know this is copied from core, so if we have a high degree of confidence it'll be removed soon, I'm not opposed to leaving it as is.